server/leafnode.go

```diff
@@ -207,6 +212,20 @@ func (s *Server) connectToRemoteLeafNode(remote *leafNodeCfg, firstConnect bool)
 	resolver := s.leafNodeOpts.resolver
 	s.mu.Unlock()
 
+	if remote != nil {
+		remote.Lock()
+		loopDelay := remote.loopDelay
```

Comment:
Would make a small func and then make `if` above `if remote != nil && remote.loopDelay() > 0`

Reply:
we need to get the value of loopDelay and reset in remote, so I don't think that this syntax is valid:
```
if remote != nil && loopDelay := remote.getLoopDelay(); loopDelay > 0 {
..
```
I could of course separate the get and reset in two functions.. still I will have to repeat remote.getLoopDelay() since again I cannot capture the value in the same `if` statement.

Reply:
ok disregard.

Reply:
Well I missed that connectToRemoteLeafNode() is invoked only with remote != nil (one of the first test in this function would return if remote is nil). So I will be able to simplify this.

Reply:
ok sounds good.

Reply:
Added a get/reset function to remote.